### DISCUSSION TITLE
Final submission doubts

### DISCUSSION THREAD

**Discussion by rithwik003** [DiveHQ/backend-internship-task#29] [Q&A] [answered]
after making the necessary changes in the new branch i have created a PR from new branch to the main branch. Should i need to merge the PR to the main brach for the assignment to get submitted.

**Comment by shuklaritvik06** [comment, marked as answer]
> after making the necessary changes in the new branch i have created a PR from new branch to the main branch. Should i need to merge the PR to the main brach for the assignment to get submitted.

Feedback would be given only on the PR#1 so you should merge your PR to the main branch for the changes to be visible in the feedback PR created by github classroom
